frontend/packages/ux-editor/src/components/Properties/PropertiesHeader/EditLayoutSetForSubform/EditLayoutSet/CreateNewSubformLayoutSet/CreateNewSubformLayoutSet.tsx

```diff
@@ -42,13 +44,14 @@ export const CreateNewSubformLayoutSet = ({
           label={t('ux_editor.component_properties.subform.created_layout_set_name')}
           value={newSubform}
           size='sm'
-          onChange={handleChange}
+          disabled={isPendingLayoutSetMutation}
+          onChange={(e) => handleChange(e.target.value)}
           error={nameError}
         />
         <StudioButton
```

Comment:
Maybe we can add the `size='sm'` to this button? It seems on the video that the button grows in size when the spinner is present. Or maybe we can make the spinner smaller on line 32? 🤔

Reply:
I agree, that looks better 🙂 I set size of the spinner to sm.

https://github.com/user-attachments/assets/7dc5ca15-3740-4205-91f0-432326635d60

Reply:
Great job!